src/CovidAppointmentTable.js

```diff
@@ -151,7 +153,28 @@ export default function CovidAppointmentTable() {
                 />
                 {ready && formattedData.length === 0 && (
                     <div role="status">
-                        <p>No appointments found.</p>
+                        <br/>
+                        <Alert severity={"info"}>
+                            <AlertTitle >
+                                No Appointments Found
+                            </AlertTitle>
+                                <p>
+                                    None of the vaccine sites that we monitor currently have available appointments. This website gathers data every minute from COVID-19 vaccine sites across Massachusetts.
```

Comment:
I'm surprised this line didn't wrap. Are you using `prettier`? If not, please make sure to run it before your code is final. Thanks!

Reply:
I set my IDE option to "Prettier on save".